x-pack/plugins/index_management/public/application/components/component_templates/component_template_list/component_template_list.tsx

```diff
@@ -183,11 +187,22 @@ export const ComponentTemplateList: React.FunctionComponent<Props> = ({
   } else if (data && data.length === 0) {
     content = <EmptyPrompt history={history} />;
   } else if (error) {
-    content = <LoadError onReloadClick={resendRequest} />;
```

Comment:
I removed this state because it didn't surface the underlying error, offered a "Try again" button which I don't think will be helpful in the typical error scenario, and was inconsistent with the other error states in this PR.